CMakeLists.txt

```diff
@@ -85,38 +92,40 @@ set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -DTOML_ENABLE_FLOAT16=0")
 # set CXXFLAGS for build targets
 set(CMAKE_CXX_FLAGS_RELEASE "-O2 -D_FORTIFY_SOURCE=2 ${CMAKE_CXX_FLAGS_RELEASE}")
 
+
 option(DEBUG_ADDRESS_SANITIZER "Enable Address Sanitizer in Debug builds" on)
 
 # If this is a Debug build turn on address sanitiser
-if (CMAKE_BUILD_TYPE STREQUAL "Debug" AND DEBUG_ADDRESS_SANITIZER)
+if ((CMAKE_BUILD_TYPE STREQUAL "Debug" OR CMAKE_BUILD_TYPE STREQUAL "RelWithDebInfo") AND DEBUG_ADDRESS_SANITIZER)
```

Comment:
everything else looks good, but i'm really not sure about this...address sanitizer is great, but does have some overhead associated with it; and it's not exactly uncommon for `RelWithDebInfo` to be used by some linux distributions or even users. imo, a better option would be to use asan by default on debug builds, while keeping it as an option for `RelWithDebInfo` or even regular `Release` builds

Reply:
I do believe that address sanitizer should not be on by default for any kind of build, but instead, explicitly activate them where needed.

Reply:
yeah this would make sense after https://github.com/PrismLauncher/PrismLauncher/pull/1518

Reply:
@getchoo could you hit Resolve so we can merge?